assets/css/main.scss

```diff
@@ -179,8 +179,14 @@ div.xoverflow {
   padding:20px
 }
 
+@keyframes highlightfade {
+    from { background: #daf8fa; }
+    to { background: transparent; }
+}
+
 :target {
-  background-color: #daf8fa;
+    animation-name: highlightfade;
+    animation-duration: 5s;
```

Comment:
nit: I think we could go a bit longer.

Reply:
I think we can also afford to make the highlighting a bit darker if it's going to fade. i tried `#8c80ff` and it looked ok to me.

Reply:
awww, that's cute. Thanks for the color swatch, github! `#ff0000` `#00ff00` `#0000ff`